Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'materialize-sass'
 gem 'font-awesome-sass'
 gem 'react-rails'
 gem 'rails_admin', '~> 1.0'
+gem 'pry-rails'
```

Comment:
I hadn't used byebug before, but you're right it does basically the same thing for debugging. I was using pry-rails mostly because it makes class instances easier to read in rails console and the debugger. But I planned on deleting it anyways, just forgot.